cmd/ui/src/views/Explore/ContextMenu/ContextMenu.test.tsx

```diff
@@ -66,7 +83,14 @@ beforeAll(() => server.listen());
 afterEach(() => server.resetHandlers());
 afterAll(() => server.close());
 
-const setup = async (permissions?: Permission[], primarySearch?: string, secondarySearch?: string) => {
+interface SetupOptions {
```

Comment:
Options object is a little more flexible and scales better, since additional params are needed now.